homeassistant/components/zwave_js/api.py

```diff
@@ -378,6 +380,7 @@ def node_status(node: Node) -> dict[str, Any]:
             cc.id == CommandClass.FIRMWARE_UPDATE_MD.value
             for cc in node.command_classes
         ),
+        "installer_mode": hass.data[DOMAIN].get(CONF_INSTALLER_MODE, False),
```

Comment:
This option isn't node specific. I don't think it should be returned per node. It should be returned as an integration wide setting. If there's no appropriate websocket command already, I suggest we add a new command.

Reply:
Thought you'd say that. Alright, I'll do it the right way then